app.rb

```diff
@@ -1,20 +1,81 @@
+require_relative "models/apartment"
+require_relative "models/tenant"
 require_relative "data"
 apartments = data[:apartments]
 tenants = data[:tenants]
 
+
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
   # Explain how the data is structured
+  # Response: the data is arranged as an object that contains two arrays which each contain hashes which contain key value pairs
+
   # What are the properties for each of the two types of hashes
+  # Response: the properties for apartment array hash are: Id, address, monthly_rent and square
+  # Response: the properties for the tenants array hash are: Id, name, age, and apartment_id
 
 # Use enumerables to -
   # Print all the addresses for the apartments
+  #   apartments.each do |apartment|
+  #   puts apartment[:address]
+  # end
+
   # Print all the names for tenants
+      # tenants.each do |name|
+      #   puts name[:name]
+      # end
+
   # Print only apartments that are less then 700 in rent
+      # for less_pay in apartments do
+      #   if less_pay[:monthly_rent] < 700
+      #     puts less_pay[:id]
+      #   end
+      # end
   # Print only tenants that are over the age of 44
+      # for mid_ten in tenants do
+      #   if mid_ten[:age] > 44
+      #     puts mid_ten[:age]
+      #   end
+      # end
   # Print only tenants that have an apartment id of 1
+      # for ten_id in tenants do
+      #   if ten_id[:id] == 1
+      #     puts ten_id[:id]
+      #   end
+      # end
   # Print all the tenants in order from youngest to oldest
+      #  arrange = tenants.sort_by do |item| item[:age]
+      #   end
+      #   puts arrange
+
+
   # Print the names of all the tenants alphabetically
+    # alpha = tenants.sort_by do |item| item[:name]
+    # end
+    # puts alpha
 
   ## More challenging
   # When printing tenants also print out the address that the tenant resides in.
+    # name_add = tenants.each do |name, address|
+    # end
+    #    if tenants.include?(tenants[:address].each)
+    #   puts "#{name}" && "#{address}"
+    # end
```

Comment:
see comment above. `name` is a confusing iterator variable if you are iterating over `tenants`

remove .each from line 60.
you're on the right track line 61, but you want the `name` and `address` properties of your iterator variable 
(you are iterating over tenants)